api/views/healthCheck/healthCheck.py

```diff
@@ -403,6 +404,147 @@ def health_check_yang_search_admin():
         return make_response(jsonify(error_response(service_name, err)), 200)
 
 
+@app.route('/yangcatalog', methods=['GET'])
+def health_check_yangcatalog():
+    service_name = 'yangcatalog'
+    status = 'running'
+    message = 'All URLs responded with status code 200'
+    additional_info = []
+
+    urls = ['http://yangcatalog.org',
+            'http://www.yangcatalog.org',
+            'https://yangcatalog.org',
+            'https://www.yangcatalog.org',
+            'http://yangvalidator.com',
+            'http://www.yangvalidator.com',
+            'https://yangvalidator.com',
+            'https://www.yangvalidator.com',
+            'http://18.224.127.129',
+            'https://18.224.127.129',
+            'http://[2600:1f16:ba:200:a10d:3212:e763:e720]',
+            'https://[2600:1f16:ba:200:a10d:3212:e763:e720]'
+            ]
+
+    for url in urls:
+        result = {}
+        result['label'] = url
+        try:
+            if any(char.isdigit() for char in url):
+                response = requests.get(url, verify=False)
+            else:
+                response = requests.get(url)
+            status_code = response.status_code
+            app.LOGGER.info('URl: {} Status code: {}'.format(url, status_code))
+            result['message'] = '{} OK'.format(status_code)
+        except:
+            result['message'] = '500 NOT OK'
+            status = 'problem'
+            message = 'Problem occured, see additional info'
+        additional_info.append(result)
+
+    return make_response(jsonify({'info': '{} is available'.format(service_name),
+                                  'status': status,
+                                  'message': message,
+                                  'additional_info': additional_info}), 200)
+
+
+@app.route('/confd-full-check', methods=['GET'])
+def health_check_confd_full_check():
+    service_name = 'ConfD - Full check'
+    status = 'running'
+    message = 'All requests responded with status code 200'
+    additional_info = []
+    check_module_name = 'confd-full-check'
+    confd_prefix = '{}://{}:{}'.format(yc_gc.protocol, yc_gc.confd_ip, repr(yc_gc.confdPort))
+    headers = {'Content-type': 'application/yang-data+json', 'Accept': 'application/yang-data+json'}
```

Comment:
please create a python file that will contain static variables
confd_header_str = 'applicaton/yang-data+json'
confd_content_type = {'Content-type': confd_header_str }
confd_accept = {'Accept': confd_header_str }
confd_headers = {confd_content_type , confd_accept }

and reuse confd_headers accross the whole project. We will update this file with any string or dict that is used multiple times so we have to do the change only on one place instead of multiple if something changes. I remember that confd headers already did once.

Reply:
done for healthCheck.py, rest will be added in another pull request